eth/backend.go

```diff
@@ -560,7 +560,7 @@ func New(ctx context.Context, stack *node.Node, config *ethconfig.Config, logger
 			heimdallClient = heimdall.NewHeimdallClient(config.HeimdallURL, logger)
 		}
 
-		if config.PolygonSync || config.PolygonSyncStage {
```

Comment:
need to integrate the bridge with a different store for the sync stage integration
will do this separately as part of https://github.com/ledgerwatch/erigon/issues/11195
for now removing since it breaks the stage integration